Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2843,21 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
```

Comment:
please put this setnence to the second line:
`Setting to false will hide mosaic button for the admin screen.`